frontend/mr-admin-flate/src/components/tilsagn/form/TilsagnFormFri.tsx

```diff
@@ -33,7 +31,7 @@ export function TilsagnFormFri(props: Props) {
     <TilsagnForm
       {...props}
       beregningInput={<BeregningInputSkjema />}
-      beregningOutput={<BeregningOutputPreview />}
+      beregningOutput={<TilsagnBeregningPreview />}
```

Comment:
Er denne linjen lik for alle varianter nå? I så fall kan det kanskje flyttes rett inn i `TilsagnForm` i stedet, så slipper den å repeteres per type